framework-cloud/framework-openshift-operator/src/main/java/org/kie/cloud/openshift/operator/scenario/builder/AbstractOpenshiftScenarioBuilderOperator.java

```diff
@@ -28,4 +29,69 @@ protected ExternalDeploymentFactory getExternalDeploymentFactory() {
         return factory;
     }
 
+    private boolean isDisableSslNull(KieApp kieApp) {
+        return kieApp.getSpec().getCommonConfig().getDisableSsl() == null;
+    }
+
+    private boolean isDisableSslTrue(KieApp kieApp) {
+        return kieApp.getSpec().getCommonConfig().getDisableSsl().booleanValue();
+    }
+
+    private boolean isDisableSslFalse(KieApp kieApp) {
+        return !isDisableSslTrue(kieApp);
+    }
+
+    protected void checkHttpKieServerRouteConfig(boolean kieServerHostnameSet, KieApp kieApp) {
+        if (kieServerHostnameSet) {
+            throw new IllegalStateException("Kie Server hostname has been already set. Update your scenario.");
+        }
+        if (!isDisableSslNull(kieApp) && isDisableSslFalse(kieApp)) {
+            throw new IllegalStateException("SSL is enabled. It is not possible to configure http hostname.");
+        }
+
+        if (isDisableSslNull(kieApp)) {
```

Comment:
not sure you need such a check.
you check already: `if not null and false`
then you can just do `kieApp.getSpec().getCommonConfig().setDisableSsl(true);` wihtout the `if`

Reply:
yes this check is needed in case disables ssl parameter is not set. This parameter is used to set up the url protocol.